task/rhacs-m2m-authenticate/0.1/README.md

```diff
@@ -0,0 +1,66 @@
+# Red Hat Advanced Cluster Security Deployment Check Task
+
+Exchanges a service account token against a short-lived authorization token to configure a pipeline run using `roxctl`.
+
+## Prerequisites
+
+This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet) or [StackRox](https://www.stackrox.io).  It also requires configuration of a machine-to-machine integration.
+
+<https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
+
+## Install the Task
+
+```bash
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-m2m-authenticate/0.1/raw
+```
+
+## Parameters
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central. Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_config_dir`**: Path to the roxctl config directory within the `roxctl-config` workspace (if machine to machine authentication is used). The path should be prefixed with `/roxctl-config`. Examples: _"/roxctl-config", **""**_.
+- `rox_image`: The image providing the roxctl tool (optional). Default: quay.io/stackrox-io/roxctl:4.4.2 (this is also the minimum version working with this task). 
+
+## Workspaces
+
+- **roxctl-config**: A [Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md) in which to write the configuration for roxctl.
+
+## Usage
+
+
+Check the [documentation](https://docs.openshift.com/acs/operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access_configure-short-lived-access) to configure the trust with the OIDC token issuer.
```

Comment:
As mentioned above this should ideally use some sample configuration to apply i.e. the k8s oidc config.